spec/support/helpers/requests_helper.rb

```diff
@@ -1,10 +1,14 @@
 module RequestsHelper
-  def log_in(username, password)
+  def log_in(username: user.email, password: user.password)
```

Comment:
I forgot to say in the last review, but this is a really nice improvement.
Nice DRY-ing out. Thanks!